src/matter/cluster/server/OnOffServer.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * @license
+ * Copyright 2022 The node-matter Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { OnOffCluster } from "../OnOffCluster";
+import { ClusterServerHandlers } from "./ClusterServer";
+
+/*
+TODO: Global Cluster fields needs to be added also here because, as discussed, based on the implementation.
+* Cluster Revision: 4 (If I get it right from the Specs - Application Cluster Specs 1.5.1)
+* FeatureMap:
+  * Bit 0 set to 0 for now because not supported
+* AttributeList:
+  * onOff
+* AcceptedCommandList:
+  * Supported always: Off, On, Toggle
+* GeneratedCommandList: empty
+* EventList: empty
+* FabricIndex: empty
+ */
+
+export const OnOffClusterHandler: () => ClusterServerHandlers<typeof OnOffCluster> = () => ({
```

Comment:
This doesn't need to be a function since there is no configuration:
export const OnOffClusterHandler: ClusterServerHandlers<typeof OnOffCluster> = {

Reply:
Yes I know, but it will be as soon as we add the next PR when we implement more logic (okk then it even need to be  a class/instance as discussed, thats why I left it that way. I can also change it. as you like